ucca/core.py

```diff
@@ -211,7 +251,7 @@ class Edge:
 
     ID_FORMAT = "{}->{}"
 
-    def __init__(self, root, tag, parent, child, attrib=None):
```

Comment:
How about optionally allowing the `tag` parameter as a shortcut for adding a single category?